packages/components/BuyerDelivery.tsx

```diff
@@ -1,5 +1,307 @@
-import { Box } from '@chakra-ui/react';
+import {
+  Box,
+  Button,
+  Collapse,
+  FormControl,
+  FormControlProps,
+  FormErrorMessage,
+  FormHelperText,
+  FormLabel,
+  FormLabelProps,
+  Input,
+  Radio,
+  RadioGroup,
+  Select,
+  Stack,
+  Text,
+  TextProps,
+} from '@chakra-ui/react';
+import { useFormik } from 'formik';
+
+import UFS from '@packages/config/ufs';
+import { EDeliveryWays } from '@packages/enums/EDeliveryWays';
+
+import InputMask, { cepMask } from './InputMask';
+
+const titleStyle: TextProps = {
+  marginTop: '1em',
+  fontSize: '25px',
+  fontWeight: 'bold',
+  color: 'black',
+};
+
+const fieldsetStyle: FormControlProps = {
+  border: '1px solid #ccc',
+  padding: '1em',
+  borderRadius: '0.3em',
+};
+
+const fieldsetLabelStyle = {
+  fontSize: '18px',
+  fontWeight: 'bold',
+  color: 'gray.600',
+};
+
+const radioLabelStyle: TextProps = {
+  fontSize: '14px',
+  fontWeight: 'bold',
+  color: 'gray.600',
+};
+
+const labelStyle: FormLabelProps = {
+  fontSize: '14px',
+  color: 'gray.600',
+};
+
+const inputStyle = {
+  borderColor: 'gray.400',
+  borderRadius: '4px',
+  bgColor: 'gray.300',
+};
+
+function isReqHasMinSize(
```

Comment:
muito boa a ideia de criar a função para evitar duplicar código 👏🏽 

Alguns pontos de melhoria:
- o nome da função não esta muito descritivo. Geralmente, a função começar com `is` indica que ela retorna um booleano, que não é o caso. O que acha de `validateFieldFilled`?
- sempre que possível, deveríamos evitar passar variáveis para serem mutadas pela função pq isso pode gerar efeitos colaterais indesejados. O que acha de retornar a msg de erro quando ela existir ao invés de adicionar uma entrada ao objeto? Poderíamos mudar a adição da entrada para o lugar em que a função é chamada.
- ao passar o objeto e a key para ser indexada, a função passar a saber a estrutura de algo que é externo a ela. Ao invés disso, poderíamos já passar `obj[key]`
- feita essas modificações, poderíamos fazer a chamada da função utilizando um `for` iterado por todas as chaves.

Reply:
Blz vou fazer a correção